json_schemas/users.json

```diff
@@ -22,6 +22,11 @@
       "type": "string",
       "pattern": "^(buyer|supplier|admin)$"
     },
+    "supplier_id": {
```

Comment:
There are a couple of underscore/camelCase anomalies I'm sorting out in the branch I'm working on at the moment.  In the interest of getting this merged I'll wrap this one up with those changes later.